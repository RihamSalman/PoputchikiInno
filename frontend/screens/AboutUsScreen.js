import React from 'react';
import { View, Text, Button, StyleSheet } from 'react-native';

const AboutUsScreen = ({navigation}) => {
    return (
      <View style={styles.container}>
        <Text>Who we are?</Text>
      </View>
    );
};

export default AboutUsScreen;

const styles = StyleSheet.create({
  container: {
    flex: 1, 
    alignItems: 'center', 
    justifyContent: 'center'
  },
});