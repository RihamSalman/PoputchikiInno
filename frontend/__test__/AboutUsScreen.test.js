jest.useFakeTimers()
import React from 'react';
import renderer from 'react-test-renderer';
import AboutUsScreen from '../screens/AboutUsScreen';

test('renders correctly', () => {
    const tree = renderer.create(<AboutUsScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });