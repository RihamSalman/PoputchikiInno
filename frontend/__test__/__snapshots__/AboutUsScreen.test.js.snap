// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
    Who we are?
  </Text>
</View>
`;
